### PR TITLE
ENG-1941 type styles

### PR DESCRIPTION
**Pull request by emuvente**

### CODE REVIEW

File: src/assets/scss/global/typography.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@import 'settings';
+
+h1 {
+	font-weight: bold;
```
Comment (mcstover) [inline src/assets/scss/global/typography.scss]: This can be declared in the header-style for "small" and will apply to h1's on all screen sizes.

Comment (emuvente) [reply src/assets/scss/global/typography.scss]: Actually, the $header-style thing currently only supports font-size, line-height, margin-top, and margin-bottom. I originally had the font-weight in there, but it was not being applied.

Comment (mcstover) [reply src/assets/scss/global/typography.scss]: ahh gotcha...sorry about that. wishful thinking

Comment (emuvente) [reply src/assets/scss/global/typography.scss]: Yeah... though we should still consider moving the header styles somewhere else so they can be edited in one place

**Comment by BoulderBrains** [comment]
Late the the party, but still looking 👀 👍

**Pull request merged**